src/components/icons/notification.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react"
+
+const Notification: FC = () => (
```

Comment:
I think you don't need a functional component for this.
Maybe just a function like:
`const Notification = () => (...)`

Reply:
Yes, I changed it 👍